ui/components/component-library/banner-base/banner-base.js

```diff
@@ -91,15 +91,15 @@ BannerBase.propTypes = {
   /**
    * Additional props to pass to the `Text` component used for the `title` text
    */
-  titleProps: PropTypes.shape(Text.PropTypes),
```

Comment:
Unfortunately using `Text.propTypes` breaks so need to update proptypes to use generic object while during our migration process